pipelines/pipeline/functions/transform/SDMX_2_0.py

```diff
@@ -205,7 +205,7 @@ def generate_editions_metadata(transformedCSV, structureXML, outputPath, metadat
 
     # dump out metadata file
 
-    with open(outputPath + pathify(dataset_title) + "_" + str(datetime.now().strftime("%Y-%m")) + "-metadata.json", "w") as outfile:
+    with open(outputPath + pathify_func(dataset_title) + "_" + str(datetime.now().strftime("%Y-%m")) + "-metadata.json", "w") as outfile:
         json.dump(editions_template, outfile, ensure_ascii=False, indent=4)
 
 
```

Comment:
All looks fine with changes made to SDMX_2_0.py as seems to be change of naming conventions and brining in the pathify_function. Only aspect that might need addressed is comment mike has made above.